docs/events/yield_events.md

```diff
@@ -61,3 +60,42 @@ def progress_example():
     )
 
 ```
+
+## Yielding Other Events
+
+Events can also yield other events. This is useful when you want to chain events together. To do this, you can yield the event handler function itself.
+
```

Comment:
```suggestion
```md alert
# Reference other Event Handler via class

When chaining another event handler with `yield`, access it via the state class, not `self`.
```
```